editor/test/editor/defold_project_search_test.clj

```diff
@@ -119,122 +127,115 @@
     (let [pattern (project-search/compile-find-in-files-regex "fOoO")]
       (is (= "fooo" (re-matches pattern "fooo"))))))
 
-(deftest make-file-resource-save-data-future-test
```

Comment:
From this point on it is mostly indentation, apart from each test case now using the shared `save-data-future`.